framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -115,6 +115,16 @@ public CFAbstractTransfer(CFAbstractAnalysis<V, S, T> analysis) {
         this.sequentialSemantics = !analysis.checker.hasOption("concurrentSemantics");
     }
 
+    // Note that the user can always forcefully turn on concurrent semantics via -AconcurrentSemantics.
+    public CFAbstractTransfer(CFAbstractAnalysis<V, S, T> analysis, boolean useConcurrentSemantics) {
```

Comment:
I've looked for `CFAbstractTransfer` but don't find an invocation of this constructor. Am I missing something?